recoverMissingLumis.py

```diff
@@ -217,86 +221,81 @@ def getFiles(datasetName, runBlacklist, runWhitelist, blockBlacklist,
     files = {}
     outputDatasetParts = datasetName.split("/")
-    print "dataset",datasetName,"parts",outputDatasetParts   
+    print "dataset", datasetName, "parts", outputDatasetParts
     try:
-        #retrieve list of blocks from dataset
+        # retrieve list of blocks from dataset
         blockNames = dbsReader.listFileBlocks(datasetName)
     except:
         raise RuntimeError("Dataset %s doesn't exist in given DBS instance" % datasetName)
-    
+
     has_parent = False
     try:
-        parents = dbsReader.listDatasetParents( datasetName )
-        if parents: has_parent=True
+        parents = dbsReader.listDatasetParents(datasetName)
+        if parents: has_parent = True
     except:
         print "Dataset with no parent"
         pass
 
-    #traverse each block
+    # traverse each block
     for blockName in blockNames:
-        #deal with white and black list.
+        # deal with white and black list.
         if blockBlacklist and blockName in blockBlacklist:
             continue
         if blockWhitelist and blockName not in blockWhitelist:
             continue
-        
-        #existing blocks in phedex
-        replicaInfo = phedexReader.getReplicaInfoForBlocks(block = blockName,
-                                                           subscribed = 'y')
+
+        # existing blocks in phedex
+        replicaInfo = phedexReader.getReplicaInfoForBlocks(block=blockName,
+                                                           subscribed='y')
         blockFiles = dbsReader.listFilesInBlock(blockName, lumis=True)
-        #has_parent = dbsReader.listBlockParents(blockName)
+        # has_parent = dbsReader.listBlockParents(blockName)
         if has_parent:
             try:
                 blockFileParents = dbsReader.listFilesInBlockWithParents(blockName)
             except:
-                print blockName,"does not appear to have a parent, even though it should. Very suspicious"
+                print blockName, "does not appear to have a parent, even though it should. Very suspicious"
                 blockFileParents = dbsReader.listFilesInBlock(blockName)
         else:
             blockFileParents = dbsReader.listFilesInBlock(blockName)
 
         blockLocations = set()
-        #load block locations
+        # load block locations
         if len(replicaInfo["phedex"]["block"]) > 0:
             for replica in replicaInfo["phedex"]["block"][0]["replica"]:
-                node = replica["node"]
-                cmsSites = siteDB.PNNtoPSN(node)
-                if type(cmsSites) != list:
-                    cmsSites = [cmsSites]
-                for cmsName in cmsSites:
-                    se = siteDB.cmsNametoSE(cmsName)
-                    blockLocations.update(se)
-                    logging.debug("cmsName %s mapped to se %s", cmsName, se)
-                logging.debug("PhEDEx node %s, cmsSites %s, blockLocations %s", node, cmsSites, blockLocations)
+                PNN = replica["node"]
```

Comment:
See the first commit only. There were 3 fixes, 2 regarding SE stuff and one regarding a new change in the ACDC Server code.